rllib/utils/actor_manager.py

```diff
@@ -115,6 +117,12 @@ def ignore_errors(self) -> Iterator[ResultOrError]:
         """Return an iterator over the results, skipping errors."""
         return self._Iterator([r for r in self.result_or_errors if r.ok])
 
+    def ignore_ray_errors(self) -> Iterator[ResultOrError]:
```

Comment:
update the docstring on how this is gonna be used, either in example or in the preamble. When would one use this and not ignore_errors?

Reply:
done